### PR TITLE
Helika removal

### PR DESCRIPTION
**Pull request by aphexyuri**

### CODE REVIEW

File: Assets/WalletConnectUnity.meta
Comment (lefarchi) [inline Assets/WalletConnectUnity.meta:1]: Do we know what this folder was for?

File: Assets/WebGLTemplates.meta
Comment (lefarchi) [inline Assets/WebGLTemplates.meta:1]: Ah i also did this in my open PR 🙈

File: Assets/Treasure/TDK-Private/Editor/TDKPackagerFull.cs [hunk 1]
```diff
@@ -31,8 +29,9 @@ public static void ExportPackage()
                 "Assets/Treasure/TDK/Resources/TDKConnectThemeData.asset",
                 "Assets/Treasure/TDK/Runtime",
 
-                // code stripping
-                "Assets/link.xml"
+                // code stripping & compilation defines
+                "Assets/link.xml",
+                "Assets/csc.rsp"
```
Comment (lefarchi) [inline Assets/Treasure/TDK-Private/Editor/TDKPackagerFull.cs:34]: Updated my open PR to include this file as well when creating the release

**Review by lefarchi — APPROVED** [review]

File: Assets/WalletConnectUnity.meta
Comment (aphexyuri) [reply Assets/WalletConnectUnity.meta:1]: @lefarchi this is now under the Thirdweb dir

File: Assets/Treasure/TDK-Private/Editor/TDKPackagerFull.cs (hunk 1 above)
Comment (aphexyuri) [reply Assets/Treasure/TDK-Private/Editor/TDKPackagerFull.cs:34]: @lefarchi shoot, apologies for the overlap here

**Pull request merged**